src/compositemark/errorbar.ts

```diff
@@ -131,15 +134,20 @@ export function normalizeErrorBar(
     tooltipEncoding
   } = errorBarParams(spec, ERRORBAR, config);
 
+  const {size, ...encodingWithoutContinuousAxisAndSize} = encodingWithoutContinuousAxis;
   const makeErrorBarPart = makeCompositeAggregatePartFactory<ErrorBarPartsMixins>(
     markDef,
     continuousAxis,
     continuousAxisChannelDef,
-    encodingWithoutContinuousAxis,
+    encodingWithoutContinuousAxisAndSize,
     config.errorbar
   );
 
-  const tick: MarkDef = {type: 'tick', orient: ticksOrient};
+  const tick: MarkDef = {
+    type: 'tick',
+    orient: ticksOrient,
+    ...(size && 'value' in size ? {thickness: size.value} : {})
```

Comment:
@domoritz @kanitw I'm not sure what is the good way for getting the size's value.

Reply:
This is fine, but you can use isValueDef type guard or something?

Reply:
Provided that you wanna keep this (see general comment)

Also, this seems like markDef.size won't be applied correctly?